lib/ansible/plugins/inventory/docker_swarm.py

```diff
@@ -22,9 +22,13 @@
     description:
         - Reads inventories from the Docker swarm API.
         - Uses a YAML configuration file docker_swarm.[yml|yaml].
+        - Plugin returns following groups od swarm nodes - I(all) - all hosts; I(workers) - all worker nodes;
+          I(managers) - all manager nodes; I(leader)v- the swarm leader node;
+          I(nonleader) - all nodes except the swarm leader.
```

Comment:
How about calling it `nonleaders` instead of `nonloader`? The other groups with more than one entry also have plural names.

Reply:
You mean just in plural form?

Reply:
Yes. There are usually many of them, if you don't have a one- or two-node cluster ;-)

Reply:
I asked because you referenced it as `nonleader` vs. `nonloader` ;)

Reply:
LOL ok :D It was apparently a bit late... ;)